app/Models/Forum/Post.php

```diff
@@ -43,6 +47,19 @@ class Post extends Model
 
     private $normalizedUsers = [];
 
+    protected static function boot()
+    {
+        parent::boot();
+
+        if (!priv_check('ForumTopicModerate')->can()) {
```

Comment:
can I do it so that every query that originates from the controller will be unscoped? otherwise I'll probably have to conditionally add `withTrashed()` to every single relevant query 😕